app/controllers/microposts_controller.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class MicropostsController < ApplicationController
+  before_action :logged_in_user, only: %i[create destroy]
+  before_action :correct_user,   only: :destroy
+
+  def create
+    @micropost = current_user.microposts.build(micropost_params)
+    if @micropost.save
+      flash[:success] = 'Micropost created!'
+      redirect_to root_url
+    else
+      @feed_items = []
+      render 'static_pages/home'
+    end
+  end
+
+  def destroy
+    @micropost.destroy
```

Comment:
@micropostがnilの可能性もあるので、
念のため

```suggestion
    @micropost&.destroy
```

にしときましょうか😊

Reply:
対応しました。

Reply:
対応しました。